integration-tests/testconfig/ccip/ccip.toml

```diff
@@ -242,14 +242,23 @@ addresses_to_fund = [
 ephemeral_addresses_number = 0
 
 [Load.CCIP.Load]
-# MessageTypeWeights corresponds with [data only, token only, message with token]
-MessageTypeWeights = [10,45,45]
+# We have 3 types of messages: Token, DataWithToken, Data
```

Comment:
It would be better to speak the same language and use the product glossary (this is how the features are named). Otherwise, it leads to confusion and drift between product and implementation. Thus, I suggest renaming to:
* Token Transfer (Token)
* Programmable Token Transfer (DataWithToken)
* Messaging (Data)